packages/pages/src/components/analytics/analytics.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import {
+  AnalyticsProvider,
+  AnalyticsScopeProvider,
+  AnalyticsContext,
+  ScopeContext,
+} from "./index.js";
+
+export default {
+  title: "components/Analytics",
+  component: AnalyticsProvider,
+  subcomponents: { AnalyticsScopeProvider },
+  decorators: [
+    (Story) => (
+      <AnalyticsProvider
+        templateData={{
+          document: {
+            name: "sample entity",
+          },
+          __meta: {
+            mode: "development",
+          },
+        }}
+        enableDebugging={true}
+      >
+        <Story />
+      </AnalyticsProvider>
+    ),
+  ],
+} as ComponentMeta<typeof AnalyticsProvider>;
```

Comment:
instead of doing a const assertion, can this be updated to have `const meta: ComponentMeta<typeof AnalyticsProvider>` and `export default meta` instead? It provides better typing this way

Reply:
removed analytics stories per @tmeyer2115 suggestion